src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -503,6 +503,7 @@
     <None Include="App.config">
       <SubType>Designer</SubType>
     </None>
+    <None Include="packages.config" />
```

Comment:
👍

Reply:
but where is the file? I don't see it in this PR - maybe it's in the git ignore ?

Reply:
Yes, this file has always been ignored for some reason.

Reply:
sorry, was your intention to add it now or should this line be removed?

Reply:
I'm not sure about why my packages.config file doesn't for this project doesn't list the other nuget packages. Can a project having nuget references do without a packages.config file?

Reply:
the new style projects do use packages.config - but the old ones should have them.
I'm assuming that all the nuget references are already referenced in other project's package.config files so they're already present and it doesn't cause an issue.